process/interface.go

```diff
@@ -416,13 +416,23 @@ type RewardsHandler interface {
 	IsInterfaceNil() bool
 }
 
-// FeeHandler will return information about fees
+// FeeHandler is able to perform some economics calculation on a provided transaction
 type FeeHandler interface {
-	MinGasPrice() uint64
-	MinGasLimit() uint64
+	SetMinGasPrice(minGasPrice uint64)
+	SetMinGasLimit(minGasLimit uint64)
```

Comment:
I think these settings should be done in the constructor. Not separate functions.

Reply:
I like simple interfaces. I prefer this without these 2 functions.

Reply:
Done, added test wrapper struct